dcs/action.py

```diff
@@ -1787,6 +1787,195 @@ def dict(self):
         return d
 
 
+class PictureAction(Action):
+
+    class HorzAlignment(Enum):
+        Left = "0"
+        Center = "1"
+        Right = "2"
+
+        def __eq__(self, other) -> bool:
```

Comment:
I'm surprised to see `__eq__` defined for these. Are these actually needed?

If they are, `other: Any` (on all of them)

Reply:
Are these needed then? Could you explain why? It looks superfluous.

Reply:
@DanAlbert, I'd like to use strong types in tests. I can remove this, but is there a way we'd be able to use strong types in tests like: self.assertEqual(m_action.horz_alignment, PictureAction.HorzAlignment.Left) -- looking it from a point of view that I load a mission in Mission Editor, see the names, like Left, and I open a test and I say "aha! understood". If I see "1" in test I have to open .miz and then say "aha! understood. Let me know what you want me to do.

Reply:
Ah, I get it now. You want https://docs.python.org/3/library/enum.html#enum.StrEnum.